helm/values.yaml

```diff
@@ -113,3 +113,21 @@ hpa:
   minReplicas: 1
   maxReplicas: 5
   targetCPUUtilizationPercentage: 80
+
+kafka-topic-creator:
+  enabled: true
+  jobName: entity-service-kafka-topics-creator
+  helmHook: pre-install,pre-upgrade
+  kafka:
+    topics:
+      entity-change-events:
+        replicationFactor: 3
+        partitions: 8
+        configs:
+          retention.bytes: 8589934592 # default = -1
+          retention.ms: 86400000 # default = 604800000 (7 days)
+          max.message.bytes: 10485760 # Allow larger messages for traces
```

Comment:
Are these values correct for a low volume topic like this? Entities are much smaller than traces, and far lower volume so seems like the partitions and maxs could be reduced.

Reply:
Fixed after discussing with @laxmanchekka and @ravisingal